demo/verycustom.html

```diff
@@ -155,7 +124,7 @@ <h1 sw-video-details-title></h1>
     </div>
    </sw-video-details>
    <sw-loading>
-    <div class="lds-dual-ring"></div>
+    <div></div>
```

Comment:
You removed the loading spinner from the demo "verycustom", but I would argue that it makes more sense to remove it from all the demos EXCEPT for verycustom, after all, "custom" is supposed to be the same as the default (just giving you a starting point to begin tweaking) whereas verycustom is supposed to be wacky and bad, showing you all the things you can do differently from the default in both markup structure and style